src/components/MoneyRequestConfirmationList.js

```diff
@@ -755,6 +755,7 @@ function MoneyRequestConfirmationList(props) {
                                     ? translate('common.error.enterMerchant')
                                     : ''
                             }
+                            rightLabel={canUseViolations && PolicyUtils.isPaidGroupPolicy(props.policy) ? translate('common.required') : ''}
```

Comment:
Let's remove `canUseViolations` from here
```suggestion
                            rightLabel={PolicyUtils.isPaidGroupPolicy(props.policy) ? translate('common.required') : ''}
```